src/main/java/com/mateacademy/model/UserModel.java

```diff
@@ -0,0 +1,35 @@
+package com.mateacademy.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+import javax.persistence.*;
```

Comment:
Please don't use a wild card in imports

Reply:
You have not resolved this comment yet.

Reply:
I resolved this after commit, so I decided to push this with next commit